### ISSUE TITLE
ModuleNotFoundError: No module named 'mmcv._ext'

### ISSUE THREAD

**Issue by xuguozhi** [MengHao666/HDR#1]

**Comment by MengHao666** [comment]
Have u installed `mmcv-full` ?

**Comment by xuguozhi** [comment]
> Have u installed `mmcv-full` ?

Yeah, I have tried mmcv and mmcn-full，it seems like pytorch environment problems. If you can provide Dockerfile, that would be fine.

**Comment by MengHao666** [comment]
> > Have u installed `mmcv-full` ?
> 
> Yeah, I have tried mmcv and mmcn-full，it seems like pytorch environment problems. If you can provide Dockerfile, that would be fine.

Sorry, we might not provide Dockerfile.  However, you could reconfigure a new local environment in your device. But hopefully, please ensure the version of mmsegmentation to be same as ours first, otherwise you couldn't use our code to conduct segmentation framework. And there might be strong constriants in  the version of mmsegmentation , mmcv, cuda and pytorch. Please take care about that.

**Comment by wenkaiH** [comment]
> _No description provided._

Have you solved this problem？ （Same problem..）

**Comment by wenkaiH** [comment]
> Have you solved this problem？ （Same problem..）（me too）

I solved this problem by installing mmcv-full=1.4.0

**Comment by dashuaizeng** [comment]
> > Have you solved this problem？ （Same problem..）（me too）
> 
> I solved this problem by installing mmcv-full=1.4.0

Hello, first of all, thank you very much for your reply. However, I still reported the same error when installing mmcv full=1.4.0. Did you install it through conda or pip，Do you need to pre install which dependencies when executing. yml files.

**Comment by dashuaizeng** [comment]
> > Have you solved this problem？ （Same problem..）（me too）
> 
> I solved this problem by installing mmcv-full=1.4.0

thanks，i do it
